### PR TITLE
Fix tv [#121516763]

### PR DESCRIPTION
**Pull request by nc-lynchj**

### CODE REVIEW

**Comment by nc-babelv** [comment]
@nc-lynchj could you put the tracker id in the title please

File: src/scripts/components/bio.jsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ const React = require('react');
 const Bio = React.createClass({
   render() {
     return (
-      <div className='bio-column'>
+      <div className='bb-r-container-content-block bio-column'>
```
Comment (nc-babelv) [inline src/scripts/components/bio.jsx]: Let's use double quotes here and throughout our component markup for consitency

File: src/scripts/components/sections.jsx [hunk 2]
```diff
@@ -2,27 +2,47 @@ const React = require('react');
 
 const Sections = React.createClass({
   render() {
+    const content = [];
+    for(let i = 0; i < this.props.sections.length; i += 2) {
```
Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: NP: space after for

File: src/scripts/components/sections.jsx [hunk 3]
```diff
@@ -2,27 +2,47 @@ const React = require('react');
...
+      const section1 = this.props.sections[i];
+      let section2;
+      if(i < this.props.sections.length - 1) {
```
Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: NP: space after if

File: src/scripts/components/sections.jsx [hunk 4]
```diff
@@ -2,27 +2,47 @@ const React = require('react');
...
+        section2 = this.props.sections[i + 1]
+      }
+      content.push(
+        <div className='bb-r-container bb-r-container-inline profile-row' key = {section1.name + section2.name}>
```
Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: Double quotes as mention before, no space between attribute ```=``` and value so
```key={section1.name + section2.name}```

Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: Let's also expand multi attribute tags, so this line would become:
```
<div
  className="bb-r-container bb-r-container-inline profile-row"
  key={section1.name + section2.name}
>
```

File: src/scripts/components/sections.jsx [hunk 5]
```diff
@@ -2,27 +2,47 @@ const React = require('react');
...
+                })
+              }
+            </div>
+          </div>
+          <div className='bb-r-container-content-block bb-r-container-content-block-align-top section' key={section2.name}>
+            <div className='bb-r-uppercase-label section-title'>{section2.name}</div>
+            <div className='bb-r-uppercase-label-border section-decor' />
+            <div className='section-items'>
+              {
+                section2.items.map((item) => {
```
Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: Are we not using lodash in the dashboard?

Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: we can also use more shorthand for this arrow function, if you are just returning one line you can omit the function brackets and return keyword, i.e.:
```
section2.item.map((item) => (
  <div 
    className="bb-p1 section-item"
    key={item}
  >
    {item}
  </div>
))

File: src/scripts/components/sections.jsx [hunk 6]
```diff
@@ -2,27 +2,47 @@ const React = require('react');
...
+          <div className='bb-r-container-content-block bb-r-container-content-block-align-top section' key={section1.name}>
+            <div className='bb-r-uppercase-label section-title'>{section1.name}</div>
+            <div className='bb-r-uppercase-label-border section-decor' />
+            <div className='section-items'>
+              {
+                section1.items.map((item) => {
+                  return (
+                    <div className="bb-p1 section-item" key={item}>{item}</div>
+                  );
+                })
```
Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: same here with arrow function shorthand

File: index.html [hunk 7]
```diff
@@ -1,10 +1,11 @@
 <html>
   <head>
     <link href="build/styles/app.css" rel="stylesheet" type="text/css" />
-    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,300,300italic' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="https://nc-uat.nextcapital.com/2016-05-17-217584.0/styles/nextcapital.min.css" type="text/css" />
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,400italic,300,300italic,600,600italic' rel='stylesheet' type='text/css'>
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main" class="bb-r-section-wrapper"></div>
```
Comment (nc-babelv) [inline index.html]: expand this tag

File: src/scripts/slideshow.jsx [hunk 8]
```diff
@@ -57,7 +57,7 @@ const SlideShow = React.createClass({
 
     return (
       <div className="slide-show">
-        <ReactCSSTransitionGroup transitionName='slide-transition' transitionEnterTimeout={2000} transitionLeaveTimeout={2000}>
+        <ReactCSSTransitionGroup transitionName='slide-transition' transitionEnterTimeout={2000} transitionLeave={false}>
```
Comment (nc-babelv) [inline src/scripts/slideshow.jsx]: expand this tag

File: src/scripts/slideshow.jsx [hunk 9]
```diff
@@ -57,7 +57,7 @@ const SlideShow = React.createClass({
...
           { currentSlide }
```
Comment (nc-babelv) [inline src/scripts/slideshow.jsx]: We should decide on a convention for these binding brackets @pattertom, ```{ spaces }``` vs ```{noSpaces}```

File: src/styles/bio_slide.scss [hunk 10]
```diff
@@ -1,114 +1,157 @@
...
-    display: flex;
-    flex-direction: row;
-    flex-wrap: nowrap;
-    justify-content: center;
-    align-items: center;
+  body {
+    margin: 0;
     height: 100%;
+    width: 100%;
+		background: #fff;
```
Comment (nc-babelv) [inline src/styles/bio_slide.scss]: fix tabs here

File: src/styles/bio_slide.scss [hunk 11]
```diff
@@ -1,114 +1,157 @@
...
+  }
+}
 
-    .photo-column {
-      flex: 0 0;
-      text-align: center;
-      margin-right: 100px;
-    }
+.bb-r-section-wrapper, .bb-r-section-container, .bb-r-section-container>.bb-r-container {
```
Comment (nc-babelv) [inline src/styles/bio_slide.scss]: let's put each of these on a new line ie:
```
.bb-r-section-wrapper,
.bb-r-section-container,
.bb-r-section-container>.bb-r-container {
  ...
}
```

File: src/styles/bio_slide.scss [hunk 12]
```diff
@@ -1,114 +1,157 @@
...
+  left: 0;
+  background: #00a5f6;
+  padding: 10px;
+  
+  .tv & {
     padding: 15px;
+  }
+  
+  .bb-uppercase-label {
+		color: #FFF;
```
Comment (nc-babelv) [inline src/styles/bio_slide.scss]: fix tabs here

File: src/scripts/slideshow.jsx (hunk 9 above)
Comment (pattertom) [inline src/scripts/slideshow.jsx]: It seems from the docs that they don't pad with spaces in this context, but I'm open to opinions

File: src/styles/bio_slide.scss [hunk 13]
```diff
@@ -1,114 +1,157 @@
...
+    margin-bottom: 15px;
+  }
+}
 
-    .section-decor {
-      width: 30px;
-      border: 1px solid #00A5FB;
-      margin: 8px 0px;
-    }
+.employee-questionnaire, .employee-bio {
```
Comment (nc-babelv) [inline src/styles/bio_slide.scss]: new line here for each selector

File: src/scripts/components/sections.jsx (hunk 5 above)
Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: with lodash:
```
_.map(section2.items, (item) => (
  <div 
    className="bb-p1 section-item"
    key={item}
  >
    {item}
  </div>
))```

File: src/scripts/slideshow.jsx (hunk 9 above)
Comment (nc-babelv) [inline src/scripts/slideshow.jsx]: Padding it is, @nc-lynchj please pad all react bindings, ie ```{ binding }``` or in a tag ```<div key={ someKey } />```

File: src/scripts/components/bio.jsx [hunk 14]
```diff
@@ -3,7 +3,7 @@ const React = require('react');
...
         {
           this.props.bio.map((par, i) => {
```
Comment (nc-babelv) [inline src/scripts/components/bio.jsx]: not in your PR I know, but _.map this

File: src/styles/bio_slide.scss (hunk 13 above)
Comment (nc-babelv) [inline src/styles/bio_slide.scss]: or not, I like new lines, but we don't have a strict convention here, use new lines if the names get rather long though

File: src/scripts/components/bio.jsx [hunk 15]
```diff
@@ -3,11 +3,11 @@ const React = require('react');
...
+      <div className="bb-r-container-content-block bio-column">
         {
-          this.props.bio.map((par, i) => {
+          _.map(this.props.bio, (par, i) => {
             return  (
-              <div className='bio-paragraph' key={'par-' + i}>{par}</div>
+              <div className="bio-paragraph" key={ "par-" + i }>{ par }</div>
             );
           })
         }
```
Comment (nc-babelv) [inline src/scripts/components/bio.jsx]: use arrow return shorthand
```
_.map(this.props.bio, (par, i) => (
  <div className="bio-paragraph" key={ "par-" + i }>{ par }</div>
));
```

File: src/scripts/components/sections.jsx [hunk 16]
```diff
@@ -1,28 +1,68 @@
...
+                  return (
+                    <div
+                      className="bb-p1 section-item"
+                      key={ item }
+                    >
+                      { item }
+                    </div>
+                  );
+                })
+              }
```
Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: same here
```
_.map(section.items, (item) => (
  <div
    ...
  >
    { item }
  </div>
));
```

File: src/scripts/components/sections.jsx [hunk 17]
```diff
@@ -1,28 +1,68 @@
...
```
Comment (nc-babelv) [inline src/scripts/components/sections.jsx]: and here

File: src/scripts/slideshow.jsx [hunk 18]
```diff
@@ -44,7 +44,7 @@ const SlideShow = React.createClass({
 
     if (currentSlide.slideType === 'bio') {
       return (
-        <BioSlide args={currentSlide.args} key={this.state.slideIndex} />
+        <BioSlide args={ currentSlide.args } key={ this.state.slideIndex } />
```
Comment (nc-babelv) [inline src/scripts/slideshow.jsx]: expand this tag

File: src/scripts/slides/bio_slide.jsx [hunk 19]
```diff
@@ -6,24 +6,38 @@ const BioSlide = React.createClass({
...
-        <div className='main-content'>
-          <div className='photo-column'>
-            <img className='bio-image' src={this.props.args.person.image} />
-            <div className='bio-name'>{this.props.args.person.name}</div>
-            <div className='bio-role'>{this.props.args.person.role}</div>
+        <div className="bb-r-container bb-r-container-inline main-content">
+          <div className="bb-r-container-content-block bb-r-container-content-block-align-center employee-name-photo photo-column">
+            <div className="bb-r-container">
+              <div className="bb-r-container-content-block">
+                <img className="image-container" src={ this.props.args.person.image } />
```
Comment (nc-babelv) [inline src/scripts/slides/bio_slide.jsx]: expand this tag

File: src/scripts/slides/bio_slide.jsx [hunk 20]
```diff
@@ -6,24 +6,38 @@ const BioSlide = React.createClass({
...
     return (
-      <div className='bio-slide slide'>
-        <div className='barnacle'>
-          <img src="images/nc-logo-white.svg" className='barnacle-logo' />
-          <div className='barnacle-text'>{this.props.args.person.area}</div>
+      <div className="bb-r-section-container bio-slide slide">
+        <div className="area-barnacle bb-theme-transparent barnacle">
+          <div className="bb-c-icon-label bb-c-icon-label-static">
+            <div className="bb-c-icon-label-content-block bb-icon bb-c-icon-label-icon bb-icon-smaller">
+              <img src="images/nc-logo-white.svg" className="barnacle-logo" />
```
Comment (nc-babelv) [inline src/scripts/slides/bio_slide.jsx]: expand this tag

**Comment by nc-babelv** [comment]
after shorthanding those arrow functions and expanding those tags, this looks good 🎉

File: src/scripts/components/sections.jsx (hunk 16 above)
Comment (nc-lynchj) [inline src/scripts/components/sections.jsx]: Seems like the return shorthand doesn't work with jsx

**Pull request merged**